File: src/main/java/ru/hlebozavod28/massak/DAO/WorkplaceCrudRepository.java
package ru.hlebozavod28.massak.DAO;

import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;
import ru.hlebozavod28.massak.domain.Workplace;

import java.util.Optional;

@Repository
public interface WorkplaceCrudRepository extends CrudRepository<Workplace, Long> {
    Optional<Workplace> findById(long id);

}

File: src/main/java/ru/hlebozavod28/massak/DAO/WorkplaceCrudRepository.kt
package ru.hlebozavod28.massak.DAO

import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository
import ru.hlebozavod28.massak.domain.Workplace
import java.util.*

@Repository
interface WorkplaceCrudRepository : CrudRepository<Workplace, Long?> {
    fun findById(id: Long): Workplace?
}